### ISSUE TITLE
Images included through the rich text do not render nicely

### ISSUE THREAD

**Issue by aquilesC** [Klimaat-Helpdesk/website#96]
Images inserted through the rich text look unstyled (they are full-width and the caption is not related to the image but is extra text). 

<img width="1282" alt="Screenshot 2021-05-03 at 09 32 31" src="https://user-images.githubusercontent.com/2494344/116851364-85ca6000-abf2-11eb-85ce-96c291931aea.png">

**Comment by daanreijnders** [comment]
I just saw that some images are not rendering correctly anymore, I think due to a style update (which looks really pretty, though!)

Here's an example: https://www.klimaathelpdesk.org/answers/is-de-productie-van-windturbines-en-zonnepanelen-schadelijk-voor-het-milieu/

![image](https://user-images.githubusercontent.com/9340879/116993210-6eb16e00-acd7-11eb-8727-c09a79f135ed.png)

**Comment by aquilesC** [comment]
Oh, yeah, I tried to make the images no higher than the view-port so it's always clear there's text after the figure and forgot to set the with to auto. Your example is actually good because it has the message about text continuing after the figure, so that problem should be gone soon.

**Comment by aquilesC** [comment]
Should be solved now, could you check, @daanreijnders ? It should also look nice on mobile. In any case, this issue is more global than the few tweaks just introduced.

**Comment by daanreijnders** [comment]
Yes, it's fixed. Thanks Aquiles! And agreed that image formatting is something to think about in the future, but this is a nice start.
